### PR TITLE
chore: update ui-kit and make login submit on enter

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

Update ui-kit to 6.1.1.

And allow login on enter (that's why I put semvar fix)

### CODE REVIEW

File: packages/application-shell/src/components/login/login.js [hunk 1]
```diff
@@ -204,16 +204,13 @@ export class Login extends React.PureComponent {
...
-                  "submit" button, making it impossible to handle "submit on enter".
-                  Once that is fixed in the underlying component, the "submit on enter"
-                  will work. */}
                   <PrimaryButton
                     label={
                       this.state.loading
                         ? this.props.intl.formatMessage(messages.validating)
                         : this.props.intl.formatMessage(messages.signin)
                     }
+                    type="submit"
```
Comment (montezume) [inline packages/application-shell/src/components/login/login.js]: this bothers me EVERYTIME I use a deployed branch 🙈

File: packages/application-shell/src/components/login-sso/__snapshots__/login-sso.spec.js.snap [hunk 2]
```diff
@@ -39,7 +39,7 @@ exports[`rendering <Formik> form elements should match layout 1`] = `
         onClick={[MockFunction]}
         size="big"
         tone="primary"
-        type="button"
+        type="submit"
```
Comment (montezume) [inline packages/application-shell/src/components/login-sso/__snapshots__/login-sso.spec.js.snap]: snapshots 🤔

Comment (tdeekens) [reply packages/application-shell/src/components/login-sso/__snapshots__/login-sso.spec.js.snap]: He's just gonna say that on every PR like a bot.

File: packages/application-shell/src/components/login-sso/login-sso.js [hunk 3]
```diff
@@ -158,11 +158,8 @@ export class LoginSSO extends React.PureComponent {
                       isDisabled={formikProps.isSubmitting}
                     />
                     <Spacings.Inline>
-                      {/* FIXME: the <PrimaryButton> current does not behave like a
-                      "submit" button, making it impossible to handle "submit on enter".
-                      Once that is fixed in the underlying component, the "submit on enter"
-                      will work. */}
                       <PrimaryButton
+                        type="submit"
```
Comment (emmenko) [inline packages/application-shell/src/components/login-sso/login-sso.js]: 🎉

**Comment by emmenko** [comment]
> that's why I put semvar fix

@montezume please use the `Type:` labels, not `Semver:`, so that this PR is included in the changelog. Thanks 🙏

**Comment by montezume** [comment]
@emmenko ah, sorry. Maybe we should remove the Semver labels then.

**Pull request merged**